docs/plugins.md

```diff
@@ -40,6 +40,10 @@ A plugin configuration can have attributes:
 
 By default, the logger plugin is enabled and configured to use the service `winston` (refer to the kuzzle-plugin-logger documentation for more information).  
 
+## Basic Auth
```

Comment:
@benoitvidis  
Or we can name the plugin "auth-passport-local", to use the same name as passport strategy ?